README.md

```diff
@@ -159,8 +159,6 @@ persist that contains the packaged Lambda function.
 
 A few option are provided to access the runner instance:
 
-1.  Provide a public ssh key to access the runner by setting \`\`.
-2.  Provide a EC2 key pair to access the runner by setting \`\`.
 3.  Access via the Session Manager (SSM) by setting `enable_runner_ssm_access` to `true`. The policy to allow access via SSM is not very restrictive.
```

Comment:
Minor: Should maybe update the numbering here

Reply:
Just checked a linter. Yes, ordered lists should start with 1.